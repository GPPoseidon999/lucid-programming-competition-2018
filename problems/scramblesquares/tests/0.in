2 2
7 3 1 5
9 8 9 3
9 8 0 4
1 4 6 0
