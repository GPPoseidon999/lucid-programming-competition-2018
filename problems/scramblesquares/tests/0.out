\ 7 /\ 9 /
5 X 33 X 8
/ 1 \/ 9 \
\ 1 /\ 9 /
0 X 44 X 8
/ 6 \/ 0 \
